install_pim/manual/crontab_tasks_ee.rst.inc

```diff
@@ -14,16 +14,16 @@ You have to add these 6 commands to your local installation:
 .. code-block:: bash
     :linenos:
 
-    php /path/to/installation/pim-enterprise-standard/bin/console pim:completeness:calculate --env=prod                                         # recalculates the products completeness
-    php /path/to/installation/pim-enterprise-standard/bin/console pim:versioning:refresh --env=prod                                             # processes pending versions
-    php /path/to/installation/pim-enterprise-standard/bin/console akeneo:rule:run --env=prod                                                    # executes rules on products
-    php /path/to/installation/pim-enterprise-standard/bin/console pimee:project:notify-before-due-date --env=prod                               # check and send notifications (TeamWork Assistant)
-    php /path/to/installation/pim-enterprise-standard/bin/console pim:asset:send-expiration-notification --env=prod                             # check and send notifications for asset date delay
-    php /path/to/installation/pim-enterprise-standard/bin/console pimee:project:recalculate --env=prod                                          # recalculate project completeness (TeamWork Assistant)
-    php /path/to/installation/pim-enterprise-standard/bin/console akeneo:batch:publish-job-to-queue franklin_insights_fetch_products --env=prod # fetch suggested data for products subscribed to Franklin Insights
+    php /path/to/installation/pim-enterprise-standard/bin/console pim:completeness:calculate --env=prod                                 # recalculates the products completeness
+    php /path/to/installation/pim-enterprise-standard/bin/console pim:versioning:refresh --env=prod                                     # processes pending versions
+    php /path/to/installation/pim-enterprise-standard/bin/console akeneo:rule:run --env=prod                                            # executes rules on products
+    php /path/to/installation/pim-enterprise-standard/bin/console pimee:project:notify-before-due-date --env=prod                       # check and send notifications (TeamWork Assistant)
+    php /path/to/installation/pim-enterprise-standard/bin/console pim:asset:send-expiration-notification --env=prod                     # check and send notifications for asset date delay
+    php /path/to/installation/pim-enterprise-standard/bin/console pimee:project:recalculate --env=prod                                  # recalculate project completeness (TeamWork Assistant)
+    php /path/to/installation/pim-enterprise-standard/bin/console pimee:franklin-insights:launch-fetch-products-job-instance --env=prod # fetch suggested data for products subscribed to Franklin Insights
 
 .. note::
-   **Note:** ``franklin_insights_fetch_products`` batch job is to be set only if you activate Franklin Insights EE feature.
+   **Note:** The command ``pimee:franklin-insights:launch-fetch-products-job-instance`` is to be set only if you activate Franklin Insights EE feature.
```

Comment:
Ok just saw that now but this command name is not good. It should not contain technical details. I know this is not your fault ;)